tests/python_client/milvus_client/test_milvus_client_insert.py

```diff
@@ -1342,5 +1344,768 @@ def test_milvus_client_insert_after_json_path_index(self, enable_dynamic_field,
+        
+        # step 4: Query the collection to check the value of nullable field
+        result = self.query(client, collection_name, filter=default_search_exp,
+                   check_task=CheckTasks.check_query_results,
+                   output_fields=[default_int32_field_name],
+                   check_items={exp_res: new_row,
+                                "with_vec": True,
+                                "pk_name": default_primary_key_field_name})[0]
+
+        assert len(result) == default_nb
+        
+        self.drop_collection(client, collection_name)
+
+    @pytest.mark.tags(CaseLabel.L1)
+    @pytest.mark.skip(reason="Partial update not ready")
+    def test_milvus_client_partial_update_various_value_to_nullable_field(self):
+        """
+        Target: test PU can successfully update various value to a nullable field
+        Method:
+            1. Create a collection, enable nullable fields
+            2. Insert default_nb rows to the collection
+            3. Partial Update the nullable field with various value
+            4. Query the collection to check the value of nullable field
+        Expected: query should have correct value
+        """
+        # step 1: create collection with nullable fields
+        client = self._client()
+        schema = self.create_schema(client, enable_dynamic_field=False)[0]
+        schema.add_field(default_primary_key_field_name, DataType.INT64, is_primary=True, auto_id=False)
+        schema.add_field(default_vector_field_name, DataType.FLOAT_VECTOR, dim=default_dim)
+        schema.add_field(default_int32_field_name, DataType.INT32, nullable=True)
+        index_params = self.prepare_index_params(client)[0] 
+        index_params.add_index(default_primary_key_field_name, index_type="AUTOINDEX")
+        index_params.add_index(default_vector_field_name, index_type="AUTOINDEX")
+        index_params.add_index(default_int32_field_name, index_type="AUTOINDEX")
+        collection_name = cf.gen_collection_name_by_testcase_name(module_index=1)
+        self.create_collection(client, collection_name, default_dim, schema=schema, 
+                               consistency_level="Strong", index_params=index_params)
+
+        # step 2: insert default_nb rows to the collection
+        rows = cf.gen_row_data_by_schema(nb=default_nb, schema=schema, skip_field_names=[default_int32_field_name])
+        self.upsert(client, collection_name, rows, partial_update=True)
+
+        # step 3: Partial Update the nullable field with various value
+        new_value = 99
+        new_row = [{default_primary_key_field_name: i, 
+                    default_int32_field_name: new_value if i % 2 == 0 else None} 
+                    for i in range(default_nb)]
+        self.upsert(client, collection_name, new_row, partial_update=True)
+
+        # step 4: Query the collection to check the value of nullable field
+        result = self.query(client, collection_name, filter=default_search_exp,
```

Comment:
i think we'd better try to filter with the new values(nulll and nullable values) to verify the filtering is working on the new values